### PR TITLE
Arm: added an absolute encoder and PID variables 

### PR DESCRIPTION
**Pull request by aebill14**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/PivotSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package frc.robot.subsystems;
```
Comment (jc3m) [inline src/main/java/frc/robot/subsystems/PivotSubsystem.java:1]: It looks like this file is already in https://github.com/Mechanical-Mustaches/2024-robot/pull/8. Remember to branch off of main when making different changes.

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java [hunk 2]
```diff
@@ -3,17 +3,39 @@
 import com.revrobotics.CANSparkLowLevel.MotorType;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkAbsoluteEncoder;
+import com.revrobotics.SparkPIDController;
 
 public class ArmSubsystem extends SubsystemBase {
-     /** Creates a new FlyWheelSubsystem. */
+     /** Creates a new ArmSubsystem. */
```
Comment (jc3m) [inline src/main/java/frc/robot/subsystems/ArmSubsystem.java:10]: Is the `ArmSubsystem` different than the `PivotSubsystem`?

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java [hunk 3]
```diff
@@ -3,17 +3,39 @@
...
 
     //These motors (wheels) will work in opposite of eachother so they can shoot the note out
     private CANSparkMax m_leftWheel = new CANSparkMax(96, MotorType.kBrushless);
     private CANSparkMax m_rightWheel = new CANSparkMax(95, MotorType.kBrushless);
```
Comment (jc3m) [inline src/main/java/frc/robot/subsystems/ArmSubsystem.java:14]: This seems like flywheel specific code, does it belong in `ArmSubsystem`?

Comment (aebill14) [reply src/main/java/frc/robot/subsystems/ArmSubsystem.java]: updated the name

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java (hunk 2 above)
Comment (aebill14) [reply src/main/java/frc/robot/subsystems/ArmSubsystem.java:10]: yes ':)

File: src/main/java/frc/robot/subsystems/PivotSubsystem.java (hunk 1 above)
Comment (aebill14) [reply src/main/java/frc/robot/subsystems/PivotSubsystem.java]: whoops :O

**Pull request merged**